src/amp/components/BodyArticle.tsx

```diff
@@ -13,6 +13,7 @@ import { WithAds } from '@root/src/amp/components/WithAds';
 import { findAdSlots } from '@root/src/amp/lib/find-adslots';
 import { getSharingUrls } from '@root/src/lib/sharing-urls';
 import { buildAdTargeting } from '@root/src/lib/ad-targeting';
+import {Epic} from "@root/src/amp/components/Epic";
```

Comment:
your IDE prettier does not seem to be working correctly here, I suspect this might be the case in the other files.

Reply:
I'll dig into my settings on IntelliJ, but this also wasn't picked up by running `yarn validate` locally. I've fixed the code style inconsistencies you flagged in commit 
b443b65.